README.md

```diff
@@ -138,6 +144,68 @@ export default App;
 
 <br/>
 
+## Core Methods
+
+For a full explanation of each function parameters and return, check the [SDK-JS documentation of the Core Methods](https://github.com/mercadopago/sdk-js/blob/main/API/core-methods.md)
+
+### getIdentificationTypes
+
+Return all the document types based on the `public_key`
+
+```javascript
+import { getIdentificationTypes } from '@mercadopago/sdk-react';
+const identificationTypes = await getIdentificationTypes();
+```
+
+### getPaymentMethods
+
+Returns a payment methods list
+
+```javascript
+import { getPaymentMethods } from '@mercadopago/sdk-react';
+const paymentMethods = await getPaymentMethods({ bin: '41111111' });
+```
+
+### getIssuers
+
+Returns a issuers list
+
+```javascript
+import { getIssuers } from '@mercadopago/sdk-react';
+const issuers = await getIssuers({ paymentMethodId: 'visa', bin: '411111111' });
+```
+
+### getInstallments
+
+Returns all installments available
+
+```javascript
+import { getInstallments } from '@mercadopago/sdk-react';
+const installments = await getInstallments({
+  amount: '1000',
```

Comment:
os parametros necessários para o `getInstallments` são apenas os `amount`, `bin` e o `paymentTypeId`.
O `locale`, não é preciso enviar aqui, isso nós pegamos do HTML do cliente, e o processingMode nós pegamos do retorno de `payment_methods`

Reply:
Inclusive o processingMode eh algo mais privado, somente sellers assessorados podem mudar do padrao 'aggregator' para 'gateway'

Reply:
Os parâmetros eu peguei direto da documentação [desse método na SDK-JS](https://github.com/mercadopago/sdk-js/blob/main/API/core-methods.md#mp-instancegetinstallmentsinstallmentsparams). Acha melhor removermos daqui?

E o `paymentTypeId` que você comentou, não está na documentação e nem nos types da sdk-js também, precisamos atualizar lá?

Reply:
Pessoal, confirmando esse assunto, o ajuste seria apagar do type as propriedades tachadas na imagem, certo?

![image](https://user-images.githubusercontent.com/104934463/234079362-6dc73de5-3d7e-40cf-8dec-340585932e16.png)

Reply:
Ale, ótimo ponto, eu tinha esquecido que o integrador pode enviar esses dados, é um pouco estranho porquê são dados que voltam de payments, mas se está assim, deve ter um bom motivo. 

Vamos deixar do jeito que você fez mesmo, quanto mais maleável a gente for, melhor...

Reply:
Boa fechou, vou manter sem alterar nada nesse ponto então.